test/e2e/upgrade/alert/alert.go

```diff
@@ -143,7 +143,7 @@ func (t *UpgradeTest) Test(f *framework.Framework, done <-chan struct{}, upgrade
 	// Invariant: No non-info level alerts should have fired during the upgrade
 	firingAlertQuery := fmt.Sprintf(`
 sort_desc(
-count_over_time(ALERTS{alertstate="firing",severity!="info",alertname!~"Watchdog|AlertmanagerReceiversNotConfigured"}[%[1]s:1s])
+count_over_time(ALERTS{alertstate="firing",severity!="info",alertname!~"Watchdog|AlertmanagerReceiversNotConfigured|AggregatedAPIDown"}[%[1]s:1s])
```

Comment:
Please make this a known violation rather than an exclusion, and add more label selectors

Reply:
Done in a965a20 -> ea88b65.  Would be good to tighten this down with minor-update `Matches`, which I'm still working on, but I dunno if we need to block on that.